app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/ExportTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.github.scribeWizTeam.scribewiz
+
+import android.util.Log
+import androidx.core.content.FileProvider
+import androidx.test.core.app.launchActivity
+import junit.framework.TestCase.*
+import org.junit.Test
+import org.junit.runner.RunWith
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.scribeWizTeam.scribewiz.activities.NavigationActivity
+import java.io.File
+
+@RunWith(AndroidJUnit4::class)
+class ExportTest  {
+
+
+    // Check if the function handles non-existing file correctly
+    @Test
+    fun exportMusicXMLFile_fileDoesNotExist() {
+        // Arrange
+        val scenario = launchActivity<NavigationActivity>()
+        scenario.onActivity { activity ->
+
+
+            // Create a temp file and immediately delete it to get a nonexistent file path
```

Comment:
Why do you create a file and delete it instead of just using a random string path?

Reply:
That was my original approach however I ran into too many issues related to permissions that I didn't want to spend the time fixing so I changed my approach to simply creating a file and deleting it.